src/main/java/me/coley/recaf/ui/MainMenu.java

```diff
@@ -70,7 +71,12 @@
 	public MainMenu(GuiController controller) {
 		// TODO: Properly managed disabled state of menu items
 		this.controller = controller;
-		//
+
+		boolean isUseSystemMenuBar = controller.config().display().useSystemMenubar;
+		// macOS menubar don't show empty items
+		boolean isEmptyMenuItemsSupported = !isUseSystemMenuBar || OSUtil.getOSType() != OSUtil.MAC;
+		setUseSystemMenuBar(isUseSystemMenuBar);
```

Comment:
I think It would be more clear what the intent is if the logic was: _"Not using the menu bar, but if we are using the bar then we must be on mac"_

> `!bar || (bar && mac)`

You can ignore this, just talking out loud here